cmd/rofl/mgmt.go

```diff
@@ -606,12 +606,15 @@ var (
 			if pubName != "" {
 				secretCfg.PublicName = pubName
 			}
+			var newSecrets []*buildRofl.SecretConfig
 			for _, sc := range deployment.Secrets {
-				if sc.Name == secretName {
-					cobra.CheckErr(fmt.Errorf("secret named '%s' already exists for deployment '%s'", secretName, roflCommon.DeploymentName))
+				if sc.Name != secretName {
+					newSecrets = append(newSecrets, sc)
+					continue
 				}
+				common.CheckForceErr(fmt.Errorf("the secret named '%s' for deployment '%s' will be replaced", secretName, roflCommon.DeploymentName))
```

Comment:
The prompt needs to make sense both for the error and warning now. Perhaps this one is better:

```suggestion
				common.CheckForceErr(fmt.Errorf("the secret named '%s' for deployment '%s' already exists", secretName, roflCommon.DeploymentName))
```